client/app/bundles/course/assessment/submission/pages/SubmissionEditIndex/SubmissionEditForm.jsx

```diff
@@ -185,6 +200,26 @@ const SubmissionEditForm = (props) => {
     </Dialog>
   );
 
+  const renderTimedAssessmentDialog = () => (
```

Comment:
Can we create a shared component for this?

Reply:
created already